modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/next-steps-template-paid-or-trial/en-US.html

```diff
@@ -0,0 +1,155 @@
+					</svg>
+
+					<h1 style=" font-size: 55px; font-weight: lighter; margin-top: 1rem; ">
+						Liferay
+					</h1>
+				</div>
+
+				<div>
+					<main>
+						<div style=" display: flex; gap: 0.75rem; justify-content: center; align-items: center; ">
+							<h2 style=" font-size: 30px; font-weight: bolder; margin-top: 2.5rem; margin-bottom: 1rem; ">
+								Next Steps
+							</h2>
+						</div>
+
+						<div style=" margin: 1rem 7rem 1rem 7rem; text-align: center; color: #54555f; ">
+							<p>
+								Congratulations on the purchase of
+								<b>[%GETAPPINFORMATION_PROJECTNAME%]</b>! You will need to
+								create a license your app before deploying to your DXP instance.
+							</p>
+
+							<p>
+								Your Order ID is:
+								<b style="color: #000">[%GETAPPINFORMATION_ORDERID%]</b>
+							</p>
+
+							<p>
+								To license your app, you can click Continue Configuration below.
+								To access the Dashboard click the button below, find your Order ID
+								and choose Create License. In either case, you must have your instance details -
+								IP address, MAC address or hostname available.
+
+							</p>
+						</div>
+
+						<div style=" padding: 10px; text-align: center; margin: 2.5rem; ">
+							<a href="[%GETAPPINFORMATION_DASHBOARDLINK%]" style=" background-color: #ffffff; padding: 0.5rem 1.5rem; border-radius: 8px; width: 10rem; text-decoration: none; height: 2.5rem; color: #000; font-weight: bold; border: #000 1px solid; margin-right: 1rem; ">Go to Dashboard</a>
+							<a href="https://console.marketplacedemo.liferay.sh/projects" style=" background-color: #0b5fff; padding: 0.5rem 1.5rem; border-radius: 8px; width: 10rem; text-decoration: none; height: 2.5rem; color: #ffffff; font-weight: bold; ">Go to Cloud Console</a>
+						</div>
+					</main>
+				</div>
+
+				<footer style=" padding: 5px; text-align: center; margin: 1rem 7rem 1rem 7rem; ">
+					<a style=" color: #0b5fff; font-weight: bold; font-size: 20px; text-decoration: none; ">Learn more about App Configuration</a>
+
+					<div style=" width: 37rem; padding: 2.5rem; display: flex; flex-direction: column; justify-content: center; ">
+						<p style=" font-size: 12px;">
+							or copy and paste this link into your browser: https://click.email1.liferay.com/u/?	qs=677a386ba405fe6c622a9af0ae0502ac75e57f5336bba4ede73e50e95e0256e12e2e1c2
+							f2567210f8f0cc6aea59084a814d60d0800647b36a5461452cd859de2
```

Comment:
What is this link?

Reply:
We need to have the link, but we'll replace this link.